src/content/web/libraries.md

```diff
@@ -4,46 +4,41 @@ short-title: Web libraries
 description: Libraries and packages that can help you write Dart web apps.
 ---
 
-The [Dart SDK][] contains [dart:html][] and other libraries
-that provide low-level web APIs.
-You can supplement or replace these APIs using web packages.
+Dart provides several packages and libraries to support
+web app development, the recommended option being [package:web][].
```

Comment:
Perhaps it's only me, but if using the `package:<name>` or `dart:<name>` syntax, I personally prefer to put it in code syntax. What do you think (here and elsewhere on the page)?

Reply:
Definitely not just you! The original page didn't code syntax any of them, so that tripped me up. Thanks for catching that!